src/app.js

```diff
@@ -1,21 +1,32 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+
+require('dotenv').config();
+
+const PORT = process.env.PORT;
+const HOSTNAME = process.env.HOSTNAME;
+
+const server = http.createServer((req, res) => {
+  const normalizedUrl = new URL(req.url, `http://${req.headers.host}`);
+  const parts = normalizedUrl.pathname.split('/').slice(1);
+  const query = Object.fromEntries(normalizedUrl.searchParams.entries());
```

Comment:
Cool! But if we have URL like that `/hello/world/123?x=1&search=some1&search=some2` your code will no working. 
Good idea to use array for this situation - in this case output muse be an array of params, like: 
```js
{
  "parts": ["hello", "world", "123"],
  "query": {
    "x": "1",
    "search": ["some1", "some2"]
  }
}
```

Think how we can implement this logic for future.